PWGHF/D2H/DataModel/ReducedDataModel.h

```diff
@@ -545,7 +629,10 @@ DECLARE_SOA_TABLE(HfCandBsConfigs, "AOD", "HFCANDBSCONFIG", //! Table with confi
 // Charm resonances analysis
 namespace hf_reso_3_prong
 {
-DECLARE_SOA_COLUMN(DType, dType, int8_t); //! Integer with selected D candidate type: 1 = Dplus, -1 = Dminus, 2 = DstarPlus, -2 = DstarMinus
+DECLARE_SOA_COLUMN(DType, dType, int8_t);                                      //! Integer with selected D candidate type: 1 = Dplus, -1 = Dminus, 2 = DstarPlus, -2 = DstarMinus
+DECLARE_SOA_COLUMN(MinNumItsClsProng, minNumItsClsProng, int);                 //! minimum value of number of ITS clusters for the decay daughter tracks
+DECLARE_SOA_COLUMN(MinNumTpcCrossedRowsProng, minNumTpcCrossedRowsProng, int); //! minimum value of number of TPC crossed rows for the decay daughter tracks
+DECLARE_SOA_COLUMN(MaxChi2Tpc, maxChi2Tpc, float);                             //! maximum value of TPC chi2 for the decay daughter tracks
```

Comment:
```suggestion
DECLARE_SOA_COLUMN(NItsClsProngMin, nItsClsProngMin, int);                 //! minimum value of number of ITS clusters for the decay daughter tracks
DECLARE_SOA_COLUMN(NTpcCrossedRowsProngMin, nTpcCrossedRowsProngMin, int); //! minimum value of number of TPC crossed rows for the decay daughter tracks
DECLARE_SOA_COLUMN(Chi2TpcMax, chi2TpcMax, float);                             //! maximum value of TPC chi2 for the decay daughter tracks
```

Reply:
Done (slightly differently, to match the names in the Analysis DataModel for track extensions)